sklearn/model_selection/_search.py

```diff
@@ -754,11 +781,62 @@ def _select_best_index(refit, refit_metric, results):
             best_index = results[f"rank_test_{refit_metric}"].argmin()
         return best_index
 
+    def _get_scorers(self, convert_multimetric):
```

Comment:
I think that we should refactor this method at some point because it will be part of all metaestimators having a CV.